README.md

```diff
@@ -612,6 +614,23 @@ SAGEMAKER_TFS_NUM_BATCH_THREADS="16"
 SAGEMAKER_TFS_MAX_ENQUEUED_BATCHES="10000"
 ```
 
+## Configurable SageMaker Environment Variables
+The following environment variables can be set on a SageMaker Model or Transform Job if further configuration is required:
+
+```bash
+# Configures the logging level for GUnicorn.
+# When looking to set this environment variable, please refer to:
+# https://docs.gunicorn.org/en/stable/settings.html#loglevel
```

Comment:
Please see if you can make it a clickable link.

Reply:
Let me know how you feel about my current readme. It removes the `please refer to` language, but I felt that would just be unnecessary "boilerplate" language.

I'm open to swapping it back if we feel that having that was actually better though. 

Thanks!

Reply:
This is okay with me.

Reply:
nit: I noticed that you are using `GUnicorn`, but usually people write `Gunicorn` unless they split the word like `Green Unicorn`. Let's move to `Gunicorn` to make it consistent ?

Reply:
Good catch. Fixed.